### ISSUE TITLE
Missing video ID

### ISSUE THREAD

**Issue by SeanPesce** [SeanPesce/DXMD-Translations#3]
There is currently ~1 in-game video that won't be translated due to a missing video ID. The video is a Breach cutscene that starts with "Investors around the world today rejoiced...". If anyone encounters this video in-game, you can assist in fixing this issue. To do so, please enable the console (`Debug=1` in the configuration file), copy the video ID when the video plays in-game, and send it to me by responding to this GitHub issue.

**Comment by Kimed** [comment]
There are two versions of video sequence with "Investors around the world today rejoiced" text. One is [[assembly:/scenes/_default/default_scene.entity].pc_resourcelibdef](0009).pc_resourcelib with video_id=31647727641083435, another [[assembly:/scenes/dlc_01/20_breach/breach_master.entity].pc_resourcelibdef](0009).pc_resourcelib with video_id=0. I don't know which one (or both) is used in game. 

Also there are more contents that have video_id=0. Full list:

[[assembly:/scenes/game/20_prague/pra_master.entity].pc_resourcelibdef](0009).pc_resourcelib: "Open blinds." "Good morning, Adam. You have one new message. " (this one is NOT duplicated elsewhere)

[[assembly:/scenes/dlc_01/20_breach/breach_master.entity].pc_resourcelibdef](0009).pc_resourcelib: "Investors around the world today rejoiced..."

[[assembly:/scenes/dlc_01/10_preorder/pre_master.entity].pc_resourcelibdef](0010).pc_resourcelib: "MISSING TEXT pra_dlc_preorder_090_cut_002_1_swhittaker" - is that your generated placeholder or theirs?

[[assembly:/scenes/dlc_01/30_dlc01/pal_master.entity].pc_resourcelibdef](0010).pc_resourcelib: "MISSING TEXT pra_049_cut_001_slaw"

**Comment by SeanPesce** [comment]
Yeah so technically all the videos with a video ID of 0 are missing their video IDs. Since I didn't have the IDs, I set them to 0 in the JSON (and the translation framework is programmed to parse this as a missing video ID). I'm not convinced that any of the missing ones are actually used in-game other than the one that starts with "Investors around the world today rejoiced..."

As for the ones with "MISSING TEXT" etc., that's real data I extracted from the game files. I believe they're just unused development placeholders, so I never attempted to find their video IDs.

**Comment by Kimed** [comment]
I'm not sure why the video is not translated in Turkish version (which, as I understand, simply overwrites the English one). When I played that cutscene in another language, it correctly caught changes I made to subtitles for video_id=31647727641083435, the same one you used to add your own lines to. Anyway, if you're still looking for this code - when I payed this video, the console showed 706f70cbfebe2b to me. Hope it helps.

**Comment by SeanPesce** [comment]
Something I hadn't noticed before: the last "real" video with a missing ID (currently ID=0 in the JSON) seems to be a duplicate of 31647727641083435. I'm not sure if there's a specific situation where it would be played instead of 31647727641083435, but in my testing it does get translated. The console shows the hex format of the video ID, so 0x706f70cbfebe2b is equivalent to 31647727641083435. Maybe it's just an extraneous entry.
